Python/Cryptocurrency_Converter/crypto_converter.py

```diff
@@ -1,110 +1,95 @@
 """we will be using coinbase API as it is public
-More info below
 https://developers.coinbase.com/api/v2?python#get-exchange-rates"""
 
 import requests
+import sys
 
 
-def exchange_rates(cryptocurrency):
-    """
-    This function gets exhange rates.
-    The coinbase API returns the JSON for an input cryptocurrency with
-    data or exhangerate of one coin to all known "normal" fiat currencies
-    like BTC(input) to INR,USD,etc.
-    """
-
-    # capitalising the cryptocurrency values as the API accepts and returns
-    cryptocurrency = cryptocurrency.upper()
-
-    # These are the cryptocurrency available currently by the API.
-    # "BTC" is for Bitcoin,"ETH" is for Ethereum and so on
-    available_cryptocurrencies = [
-        'BTC', 'ETH', 'ETC', 'BCH', 'LTC', 'ZEC', 'ZRX']
-
-    if cryptocurrency not in available_cryptocurrencies:
-        return None
+def exchange_rates(typee):
```

Comment:
`typee` parameter can be renamed to something more meaningful eg. `exchange_type`